python/lsst/meas/extensions/scarlet/scarletDeblendTask.py

```diff
@@ -901,7 +910,7 @@ def deblend(self, mExposure, catalog):
                 t0 = time.monotonic()
                 # Build the parameter lists with the same ordering
                 if self.config.version == "lite":
-                    blend, skippedSources, skippedBands = deblend(
+                    blend, skippedSources, skippedBands, spectrumInit = deblend(
```

Comment:
Shouldn't the code do something differently if `spectrumInit` is `False`? Or does something else below raise an exception that gets handled?

Reply:
It's not an error. Some blends cannot run the algorithm because they are too big and the node runs out of memory. So in the code below that updates source records it just sets a column in the catalog that notifies us that it skipped spectrum initialization.

Reply:
Looking at the entire code and following along the variable makes sense now.